modules/photons_control/device_finder.py

```diff
@@ -704,11 +802,26 @@ async def matches(self, sender, fltr, collections):
 
         async def gen(reference, sender, **kwargs):
             for e in self.points_from_fltr(fltr):
+                if e == InfoPoints.VERSION:
```

Comment:
?

Reply:
This is (I'd guess) attempt 1 billiionth and 3 at getting the futures to work properly such that the information retrieval test didn't run into oblivion sending the same packets over and over. There is possibly a more elegant way to do it without breaking the InfoPoints enum, but I couldn't find it at the time.